offer-manager-domain/src/main/groovy/life/qbic/business/persons/affiliation/Affiliation.groovy

```diff
@@ -120,4 +123,20 @@ class Affiliation {
     void setCategory(AffiliationCategory category) {
         this.category = category
     }
+
+    protected boolean getIsActive() {
+        return isActive
+    }
+
+    boolean isActive() {
+        return getIsActive()
+    }
+
+    void inactivate() {
+        setIsActive(false)
+    }
```

Comment:
why do we have duplicate getters and setters here?

Reply:
yes, that is on purpose. The `getIsActive()` ist the getter necessary for JPA frameworks (such as Hibernate) to work (bean naming convention for fields). 

However, i found this method very ugly to expose to a client, a more intuitive naming scheme is to form a question to the object `isActive()` than `getIsActive()`. So this is why I introduced the public polished method :)

Reply:
I think it's confusing to have both. I personally would prefer to then just keep the one for the JPA

Reply:
but by defining the scope, the client only sees the one supposed to use

Reply:
I can add some documentation to the methods for the developers if you like, but the soulution is more client friendly imo

Reply:
I think adding documentation would be very good